### PR TITLE
Replace RanChangeSetSerializer with wrapper class for serde compatibility

### PR DESCRIPTION
**Pull request by yawkat**

### CODE REVIEW

File: liquibase/src/main/java/io/micronaut/liquibase/endpoint/RanChangeSetWrapper.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import io.micronaut.core.annotation.Introspected;
+import liquibase.change.CheckSum;
+import liquibase.changelog.ChangeSet;
+import liquibase.changelog.RanChangeSet;
+
+import java.time.Instant;
+import java.util.Set;
+
+@Introspected
+final class RanChangeSetWrapper {
```
Comment (graemerocher) [inline liquibase/src/main/java/io/micronaut/liquibase/endpoint/RanChangeSetWrapper.java:28]: Add `@Internal` and `@since`

**Review by graemerocher — CHANGES_REQUESTED** [review]

**Review by graemerocher — APPROVED** [review]

File: liquibase/src/main/java/io/micronaut/liquibase/endpoint/RanChangeSetSerializer.java [hunk 2]
```diff
@@ -1,64 +0,0 @@
...
-
-/**
- * Jackson serializer for {@link RanChangeSet}.
- *
- * @author Iván López
- * @since 1.0.0
- */
-@Singleton
-@Type(RanChangeSet.class)
-public class RanChangeSetSerializer extends JsonSerializer<RanChangeSet> {
```
Comment (ilopmar) [inline liquibase/src/main/java/io/micronaut/liquibase/endpoint/RanChangeSetSerializer.java:36]: Removing a public class is a breaking change. I doubt that someone is using it... but :shrug:

**Review by ilopmar — CHANGES_REQUESTED** [review]
The output is not the same as before (image on the right). You can see that there are a few missing fields and the name of `checksum` is now `lastChecksum`:

image

**Comment by yawkat** [comment]
Thanks, i've added a test to verify that all properties are present as they should be, and fixed it.

Also readded the serializer class, it doesn't hurt serde as long as nobody uses it.

**Review by ilopmar — APPROVED** [review]
Looks good. The only thing I'm not really sure about if these changes need to go in `5.0.1` or in `5.1.0`.

@jameskleeh thoguths?

**Comment by jameskleeh** [comment]
Deprecations can only be done in minor releases

**Comment by ilopmar** [comment]
@yawkat please change version to `5.1.0` and target micronaut core `3.3.x`

**Comment by yawkat** [comment]
done

File: liquibase/src/main/java/io/micronaut/liquibase/endpoint/RanChangeSetWrapper.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+import liquibase.changelog.RanChangeSet;
+
+import java.time.Instant;
+import java.util.Set;
+
+/**
+ * Serialization wrapper around {@link RanChangeSet}.
+ *
+ * @author Jonas Konrad
+ * @since 5.0.1
```
Comment (ilopmar) [inline liquibase/src/main/java/io/micronaut/liquibase/endpoint/RanChangeSetWrapper.java:33]: @yawkat you missed this `@since` :pray:

**Comment by yawkat** [comment]
nice catch :)

**Pull request merged**